product-service/handlers/types.ts

```diff
@@ -0,0 +1,7 @@
+export type Product = {
```

Comment:
The product model is missing an image URL and image title.

Reply:
At this moment from api we not receive image url and title, therefore they not presented at this type